frontend/src/components/PageController.tsx

```diff
@@ -4,9 +4,10 @@ import LoginPage from "./LoginPage";
 import LobbyPage from "./LobbyPage";
 import { connect } from "react-redux";
 import GamePage from "./GamePage";
+import StatsPage from "./StatsPage";
 
 interface Props {
-  state: "login" | "lobby" | "game";
+  state: "login" | "lobby" | "game" | "stats";
```

Comment:
You could extract `state` from `root-schema` and use that instead. This way, we will avoid discrepancies that could easily pop up (and did in this code :) )